templates/core/home.html

```diff
@@ -14,7 +14,7 @@ <h1 class="brand">{{ "cubes" | fas(fw=true) }} Docs.rs</h1>
 
             <div class="buttons">
                 <button type="submit" class="pure-button pure-button-normal">Search</button>
-                <button type="submit" class="pure-button pure-button-normal" id="i-am-feeling-lucky-button">
+                <button type="submit" class="pure-button pure-button-normal" id="i-am-feeling-lucky-button" name="i-am-feeling-lucky" value="1">
```

Comment:
I think you can remove the `id` now that you're not using `getElementById`.
```suggestion
                <button type="submit" class="pure-button pure-button-normal" name="i-am-feeling-lucky" value="1">
```